.meteor/packages

```diff
@@ -7,3 +7,5 @@
 meteor-platform
 mantle:router
 mantle:accounts
```

Comment:
we should remove the mantle:accounts package from the main app - I did this on the other testing branch